google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITSystemTest.java

```diff
@@ -488,6 +489,35 @@ public void greaterThanQuery() throws Exception {
     assertEquals(2L, querySnapshot.getDocuments().get(0).get("foo"));
   }
 
+  @Test
+  public void multipleInequalityQueryOnSameProperties() throws Exception {
+    addDocument("foo", 1);
+
+    QuerySnapshot querySnapshot =
+        randomColl.whereGreaterThan("foo", 0).whereLessThanOrEqualTo("foo", 2).get().get();
+    assertEquals(1, querySnapshot.size());
+    assertEquals(1L, querySnapshot.getDocuments().get(0).get("foo"));
+  }
+
+  @Test
+  public void multipleInequalityQueryOnDifferentProperties() throws Exception {
+    assumeFalse(
+        "Skip this test when running against emulator",
```

Comment:
Please explain _why_ this test is being skipped when running against the emulator. For example, append something like ", because ..." to the message.